assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -170,18 +192,39 @@ export default function SettingsView() {
 								<DisplaySetting value={ ga4MeasurementID } />
 							</p>
 						</div>
+						<div className="googlesitekit-settings-module__meta-item">
+							<h5 className="googlesitekit-settings-module__meta-item-type">
```

Comment:
Not sure how I missed this before, but this is duplicating the output for the Measurement ID
![image](https://user-images.githubusercontent.com/1621608/127973197-47c18ac7-5632-4094-a6e5-479472c9f291.png)

Reply:
Fixed now!

![oops](https://user-images.githubusercontent.com/8496063/127992120-af5959fe-2109-4956-b562-93f807a08b94.png)

That's kind of funny. It even has screenshots with the duplicate Measurement Id and we still didn't spot it  :stuck_out_tongue: